src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEMethodSymbol.vb

```diff
@@ -75,6 +76,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             Private Const s_isConditionalAttributePopulatedBit As Integer = 1 << 11
             Private Const s_isInitOnlyBit = 1 << 12
             Private Const s_isInitOnlyPopulatedBit = 1 << 13
+            Private Const s_isUnmanagedCallersOnlyBit = 1 << 14
```

Comment:
>Private Const s_isUnmanagedCallersOnlyBit = 1 << 14 [](start = 12, length = 51)

I do not think we need this bit.

Reply:
> I do not think we need this bit.

Marked as resolved, but it doesn't look addressed.

---
In reply to: [567035007](https://github.com/dotnet/roslyn/pull/50886#discussion_r567035007) [](ancestors = 567035007)